README.md

```diff
@@ -1,17 +1,24 @@
 # THEOplayerSDK
 
-THEOplayer is the universal video player solution created by THEO Technologies. It enables you to quickly deliver content playback on Android, Android TV and Fire TV.
+THEOplayer is the universal video player solution created by [THEO Technologies](https://www.theoplayer.com/). The THEOplayer Android SDK enables you to quickly deliver content playback on Android, Android TV and Fire TV.
 
 ## Prerequisites
 
--   A valid THEOplayer license. Request yours via our [THEOportal](https://portal.theoplayer.com).
+### THEOplayer license
+You must have a valid THEOplayer license. Request your license via [https://portal.theoplayer.com](https://portal.theoplayer.com) where you can create a THEOplayer Android SDK, Android TV SDK or Fire TV SDK.
+After creating an Android SDK (or Android TV SDK or Fire TV SDK), you can copy its license string to your clipboard as demonstrated in the screenshot below.
+You must use this license string when setting up THEOplayer.
+
+![](https://cdn.theoplayer.com/images/git/theoplayer-android-sdk-license-string.png)
+
 
 ## Included features
 
-THEOplayerSDK consists of modular features. This package includes only the basic features: DASH, HLS, LL-HLS and UI.
+The THEOplayer SDK consists of modular features. This package includes only the basic features: DASH, HLS, LL-HLS and UI.
```

Comment:
We also added Exoplayer and Cache at the last minute but looks like we forgot to update the documentation.
Can you please update it?

Reply:
Should be added. Can you review?